src/main/java/com/dev/careers/domain/HttpStatusConstants.java

```diff
@@ -8,4 +8,6 @@
         HttpStatus.BAD_REQUEST);
     public final static ResponseEntity<Void> CREATE = new ResponseEntity<Void>(
         HttpStatus.CREATED);
+    public  final static ResponseEntity<Void> OK = new ResponseEntity<>(HttpStatus.OK);
```

Comment:
checkstyle 빨리 논의하셔서 적용하시면 좋을 것 같네요~

```suggestion
    public final static ResponseEntity<Void> OK = new ResponseEntity<>(HttpStatus.OK);
```